src/tmxrasterizer/tmxrasterizer.cpp

```diff
@@ -32,9 +32,11 @@
 #include "imagelayer.h"
 #include "isometricrenderer.h"
 #include "map.h"
+#include "mapformat.h"
```

Comment:
You don't seem to need this include.

Reply:
Hmm, the build fails for me without it (MapReader.readMap doesn't seem to look for all readers, so I'm using the readMap from mapformat.cpp).  That said, now that you mention this, I should remove the now-unnecessary MapReader instances that get created.

Reply:
Whoops, I didn't realize I put `readMap` in `mapformat.h`!